submissions/MosQuitO404/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Hooli-style-Popup</title>
+</head>
+
+<body>
+    <header class="popup-header">
+        <nav class="nav-items">
+            <ul class="nav-items-ul">
+                <li class="nav-items-li">
+                    <a class="nav-items-element" href="#">Home</a>
+                </li>
+                <li class="nav-items-li">
+                    <a class="nav-items-element" href="#">About</a>
+                </li>
+                <li class="nav-items-li">
+                    <a class="nav-items-element" href="#">Pricing</a>
+                </li>
+                <li id="popup-wrapper">
+                    <input type="checkbox" id="central-button">
+                    <label id="central-button-label" for="central-button"><img src="sources/icons/free-icon-menu-7512756.png" id="central-button-logo" alt=""></label>
+                    <div id="popup-section">
+                        <div class="popup-set">
+                            <ul class="popup-set-ul">
+                                <li class="popup-set-li">
+                                    <span>
+                                        <a class="popup-element" href="#"><img src="sources/icons/free-icon-house-3845763.png" class="icons-pictures" alt=""><br>Home</a>
+                                    </span>
+                                </li>
+                                <li class="popup-set-li">
+                                    <span>
+                                        <a class="popup-element" href="#"><img src="sources/icons/free-icon-contacts-3845847.png" class="icons-pictures" alt=""><br>Contacts</a>
```

Comment:
In the future avoid using `<br>` for defining block in new line)

Reply:
Ok, thanks 🙏☺️ I understand 😌